packages/terra-navigation-side-menu/src/MenuItem.scss

```diff
@@ -1,4 +1,11 @@
 :local {
+  .list-item {
```

Comment:
These are needed so we can apply borders to the side on one element and bottom on another element without creating a diagonal border effect (This occurs when both top and/or bottom borders are set with side borders). Only the bottom border is needed for the legacy theme, though I've included all border directions to match consistency with the other styles in this file.